src/ProjectList/defaultProjects.ts

```diff
@@ -34,34 +28,44 @@ export const defaultProjects: Project[] = [
       { title: "Staging", target: "https://stagingapi.artsy.net" },
       { title: "Production Sidekiq", target: "https://api.artsy.net/admin/sidekiq" },
       { title: "Staging Sidekiq", target: "https://stagingapi.artsy.net/admin/sidekiq" },
-      { title: "GitHub", target: "https://github.com/artsy/gravity", primaryAction: true },
-      { title: "CircleCI", target: "https://app.circleci.com/pipelines/github/artsy/gravity", secondaryAction: true },
+      { title: "Sentry Production", target: "https://artsynet.sentry.io/issues/?project=157936" },
+      { title: "Sentry Staging", target: "https://artsynet.sentry.io/issues/?project=157935" },
     ],
   },
   {
     name: "Metaphysics",
     links: [
       { title: "Production", target: "https://metaphysics-production.artsy.net" },
       { title: "Staging", target: "https://metaphysics-staging.artsy.net" },
-      { title: "GitHub", target: "https://github.com/artsy/metaphysics", primaryAction: true },
-      {
-        title: "CircleCI",
-        target: "https://app.circleci.com/pipelines/github/artsy/metaphysics",
-        secondaryAction: true,
-      },
     ],
   },
+  { name: "Palette", links: [] },
+  { name: "Palette-Mobile", links: [] },
   {
     name: "Volt",
     links: [
       { title: "Production", target: "https://cms.artsy.net" },
       { title: "Staging", target: "https://cms-staging.artsy.net" },
-      { title: "GitHub", target: "https://github.com/artsy/volt", primaryAction: true },
+      { title: "Sentry Production", target: "https://artsynet.sentry.io/issues/?project=1281370" },
+      { title: "Sentry Staging", target: "https://artsynet.sentry.io/issues/?project=1281353" },
```

Comment:
Thought Sentry links might be useful :)

Reply:
Yeah I wondered about that - sounds good to me!